packages/components/Separator/src/Separator.types.ts

```diff
@@ -1,26 +1,27 @@
 import { ViewProps } from 'react-native';
-import { IForegroundColorTokens } from '@fluentui-react-native/tokens';
-import { IViewWin32Props } from '@office-iss/react-native-win32';
 
-export const separatorName = 'RNFSeparator';
-
-interface ISeparatorPropTokens {
+interface SeparatorPropTokens {
   /**
    * Specifies whether the separator is vertical or horizontal. If true, then it's vertical.
    * @defaultValue 'false'
    */
   vertical?: boolean;
 }
 
-export interface ISeparatorTokens extends IForegroundColorTokens, ISeparatorPropTokens {
+export interface SeparatorTokens extends SeparatorPropTokens {
   /**
    * Specifies the width of the separator. This will be interpreted as DIPs on Windows and Android, but as points on Mac and iOS.
    * @defaultValue 1
    */
   separatorWidth?: number;
+
+  /**
+   * Specifies the color of the separator
+   */
+  color?: string;
```

Comment:
Did you see the tokenset stuff the web folks were doing, out of curiosity?

Reply:
I just looked at it now, it looks like there are two things if I'm reading it right:

- a naming convention for the equivalent of our things like `FontTokens` where they should be named `FontTokenSets` which sounds reasonable and something we should align on.
- the ability to override tokens via props. I'm less aligned on this one. My read is that this is happening because of the move away from having compose and customize as options. 

For the second one we could handle this, with the memo stuff we do it could be blended as an object reference fairly easily. But it seems like something that can be added later if necessary. For now I would hold off.

For the first one with the naming we should snap to that. I'm also temped to move the types into theme-types as they are sort of global theme/customization fragments.

Reply:
Note that in this case we had IForegroundColorTokens which always just contained color?: string and I put it in specifically to make it more readable, but maybe it is better to keep these composed of parts.